pkg/virt-controller/watch/descheduler/descheduler.go

```diff
@@ -19,7 +19,85 @@
 
 package descheduler
 
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/client-go/tools/cache"
+	virtv1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/controller"
+
+	k8sv1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/client-go/log"
+
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+)
+
 // EvictOnlyAnnotation indicates pods whose eviction is not expected to be completed right away.
 // Instead, an eviction request is expected to be intercepted by an external component which will initiate the
 // eviction process for the pod.
 const EvictOnlyAnnotation = "descheduler.alpha.kubernetes.io/request-evict-only"
+
+// EvictionInProgressAnnotation indicates pods whose eviction was initiated by an external component.
+const EvictionInProgressAnnotation = "descheduler.alpha.kubernetes.io/eviction-in-progress"
+
+func AddDeschedulerAnnotation(virtClient kubecli.KubevirtClient, sourcePod *k8sv1.Pod) error {
```

Comment:
nit: I'd call this `descheduler.MarkEvictionInProgress()', it comes from the descheduler package so I don't think you have to repeat that in the function name (at least in this case).

Reply:
Nice suggestion, I moved it without renaming it. :+1: